os/kernel/binary_manager/binary_manager_recovery.c

```diff
@@ -207,6 +207,15 @@ void binary_manager_exclude_rtthreads(struct tcb_s *tcb)
 	prev = tcb->bin_blink;
```

Comment:
For readability and better understanding, Let me suggest another proper name of this function.
binary_manager_exclude_rtthreads(tcb); ==> binary_manager_deactivate_rtthreads(tcb);